### ISSUE TITLE
Switch to student account fails

### ISSUE THREAD

**Issue by aidenZ1193** [dailinshen/coms4156_jumpstart#8]
In one of the unit tests, we want to open attendance window for one class and have the student account log in to it. But it fails on assertIn("COMS4111", rv.data), which should pass since the student has been added to the class.

**Comment by aidenZ1193** [comment]
To figure the problem, I first put a pdb.set_trace() after the student has been added to the class. So when the tests pause on pdb, I open the webpage on my local machine and logged in using that student account. The class "COME4111" does shows on the student's page, which is as expected. 
Hence what it has proved is that when we switch to the student's account, it didn't actually worked as we thought, though it didn't raise an error. To find out which account we were still using, Dailin added a new feature to the student and teacher's html page, to make it shows the UNI of it's account. Teacher's account won't show UNI, just the student's page will.

**Comment by aidenZ1193** [comment]
After we can see the uni, we run the test again and noticed that when trying to signin to attendance window, we were still at teacher's account. After I did some digging to our test code, I found that we forget to make teacher log out in the unittests we ran before. 
So we added logout code to the unittests that runs before test_student(), we successfully found the student's uni on it's webpage information, which proved that we have logged out from teacher's account and re-logged in to a student's account. And after that, the test_student() passed.
